backend/server/router.go

```diff
@@ -0,0 +1,43 @@
+package server
+
+import (
+	"github.com/gorilla/mux"
+	"net/http"
+	"peoplemath/storage"
+	"time"
+)
+
+const (
+	DefaultStoreTimeout = 5 * time.Second
+)
+
+// Server struct to handle incoming HTTP requests
+type Server struct {
+	store        storage.StorageService
+	storeTimeout time.Duration
+}
+
+func InitServer(store storage.StorageService, storeTimeout time.Duration) *Server {
+	return &Server{
+		store:        store,
+		storeTimeout: storeTimeout,
+	}
+}
+
+func MakeHandler(s *Server) http.Handler {
```

Comment:
Any reason we can't just make this a method on the server, like it was originally?

`func (s *Server) MakeHandler() {...`

Reply:
Hi, sorry for the slow response on this! This method is just for separate routes from main.go. Do you have an idea to leave creating handlers in main.go? If so - it would not be a problem for me to move them back where they were initially.